test/zdtm/static/ghost_holes_large00.c

```diff
@@ -0,0 +1,148 @@
+#include <errno.h>
+#include <stdbool.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <string.h>
+#include <linux/limits.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc = "Test ghost with one large hole(10Gi) in the middle";
+const char *test_author = "Liang-Chun Chen <featherclc@gmail.com>";
+
+char *filename;
+TEST_OPTION(filename, string, "file name", 1);
+
+/* Buffer that is suitable for hole size */
```

Comment:
minor nit: I believe we need to reword "hole" -> "data", looks like a leftover.

Reply:
OK, I have fixed them.